app/Http/Controllers/AiConvController.php

```diff
@@ -120,6 +120,34 @@ public function removeConv(Request $request, $slug){
         return response()->json(['success' => true, 'message' => 'Conv deleted successfully']);
     }
 
+
+    public function editConvTitle(Request $request, $slug){
+        $user = Auth::user();
+        $conv = AiConv::where('slug', $slug)->firstOrFail();
+
+        // Check if the conv exists
+        if (!$conv) {
+            return response()->json(['success' => false, 'message' => 'Conv not found'], 404);
+        }
+
+        // Check if the user is an admin of the conv
```

Comment:
```suggestion
        // Check if the user is the owner of the conv
```

Just to not confuse with "admin" in other contexts